ios/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge/RCTGoogleAnalyticsBridge.m

```diff
@@ -138,6 +138,14 @@ - (NSDictionary *)constantsToExport
                                                         target:targetUrl] build]];
 }
 
+RCT_EXPORT_METHOD(setCustomDimensionValue:(NSString *)trackerId dimensionIndex:(NSInteger)dimensionIndex dimensionValue:(NSString *)dimensionValue)
+{
+    id<GAITracker> tracker = [[GAI sharedInstance] trackerWithTrackingId:trackerId];
+    
+    [tracker set:[GAIFields customDimensionForIndex:dimensionIndex] value:dimensionValue];
```

Comment:
Yes that is also a valid way of handling it, since it is the same tracker instance. In that case we should probably specify in the readme that the custom dimension values will not be sent unless/until you track a screen/event. I'm fine with handling it like that :)